e2e/plugin-typescript-e2e/mocks/fixtures/default-setup/code-pushup.config.ts

```diff
@@ -0,0 +1,22 @@
+import type { CoreConfig } from '@code-pushup/models';
+import {
+  getCategoryRefsFromGroups,
+  typescriptPlugin,
+} from '@code-pushup/typescript-plugin';
+
+export default {
+  plugins: [
+    await typescriptPlugin({
+      tsConfigPath: 'tsconfig.json',
+    }),
+  ],
+  categories: [
+    {
+      slug: 'typescript-quality',
+      title: 'Typescript',
+      refs: await getCategoryRefsFromGroups({
+        tsConfigPath: 'tsconfig.json',
+      }),
+    },
+  ],
```

Comment:
Wasn't `tsConfigPath` renamed? I guess you can omit it anyway, since `tsconfig.json` is the default value.